restapi.go

```diff
@@ -233,26 +235,24 @@ func (self *RestAPI) pushNotification(reqId string, kv map[string]string, perdp
 		case "service":
 			// three keys need to be ignored
 		case "badge":
-			if v != "" {
-				var e error
-				_, e = strconv.Atoi(v)
-				if e == nil {
-					notif.Data["badge"] = v
-				} else {
-					notif.Data["badge"] = "0"
-				}
+			if _, err := strconv.Atoi(v); err == nil {
+				notif.Data["badge"] = v
+			} else {
+				notif.Data["badge"] = "0"
 			}
 		default:
 			notif.Data[k] = v
 		}
 	}
 
+	logger.Debugf("Parameters accepted: %+v", notif)
+
 	if notif.IsEmpty() {
-		logger.Errorf("RequestId=%v From=%v Service=%v EmptyNotification", reqId, remoteAddr, service)
+		logger.Errorf("RequestId=%v From=%v Service=%v NrSubscribers=%v Subscribers=\"%+v\".", reqId, remoteAddr, service, len(subs), subs, err)
 		return
 	}
 
-	logger.Infof("RequestId=%v From=%v Service=%v NrSubscribers=%v Subscribers=\"%+v\"", reqId, remoteAddr, service, len(subs), subs)
+	logger.Infof("RequestId=%v From=%v Service=%v NrSubscribers=%v Subscribers=\"%+v\".", reqId, remoteAddr, service, len(subs), subs)
```

Comment:
Oh, English is my second language too, I see now. This explains all those `.`s after our pushtypes in our logs. I was wondering if that was something that was from the original code. Can we remove it please?